src/transport/include/antioch/mixture_averaged_transport_evaluator.h

```diff
@@ -398,11 +400,12 @@ namespace Antioch
                                                                   mu[s] );
           }
 
-        k_mix += k[s]*chi[s]/phi_s;
+        k_sum1 += k[s]*chi[s];
+	k_sum2 += chi[s]/k[s];
         mu_mix += mu[s]*chi[s]/phi_s;
 
       }
-
+    k_mix = .5*(k_sum1+1/k_sum2);
```

Comment:
We need this better commented.  Of course, we needed the *previous* version much better commented...

Why is this new formula appropriate for k but not mu?

Reply:
Ok, works for me then.  Still curious why it would be an improvement for conductivity but not viscosity, though.

Reply:
I'm not sure really, the reacting flow book I obtained it from, basically just stated this formula for the conductivity ontop of the normal viscosity one. I'll look into it more to see if I can obtain a better reasoning.

Reply:
There are a class of empirical averaging formulas(the Kee formula is a version of one of these) that are less accurate(but much cheaper) than the Wilke formula.   There is a long discussion of this in 
A. Ern and V. Giovangigli, Multicomponent Transport Algorithms, Lecture Notes in Physics, New Series Monographs, m 24, (Springer-Verlag, Heidelberg, 1994).

I don't know why Sylvain implemented the Wilke rule for thermal conductivity(I told him to use the Kee formula ala the Chemkin manual for consistency).  Both Chemkin and Cantera use the Kee empirical rule.

Reply:
Hello all,

Well I won't be able to remember the whole situation, but it seems to me that there was lots of considerations in this part.  Isn't it also the part where there was an implementation given by a french fortran code?  I remember using their tricks to deal with Titan's high atmosphere (like huge amount of N2, little of CH4 and traces of what we want to follow), and theirs was the only one that would not give the numerical equivalent of kicking the ground in fury, then sit down while breaking in tears...  But I cannot certify it was this exact part of the code.

Might worth a try to look for that code, they might have left a better documentation than me...

Now there is also the argument that I totally prefer rules (like Wilke) than empirical, but that might be because DFT looks sometimes like a bad magic trick to me (Chris can very probably explain what that means).  In that case, my bad, I'm sorry.  There is also the possibility that it was just before I left and I did not do it because I did not find the time before leaving...

Reply:
It's entirely possible @SylvainPlessis did the Kee formula originally and I changed it because I didn't know any better. (I don't have time to troll through the history). Anyway, I don't want @SylvainPlessis to feel like he's getting dumped on because that's not my intention.

Besides I haven't gotten to speak to either @SylvainPlessis or @variscarey in forever! :(

The French code is EGLib (Ern, Giovangigli) is still available. It's in Fortran. :(